api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -656,6 +660,17 @@ private void enableConsultInitial() {
                 sessionLocationHasTag("Consult Note Location")));
     }
 
+	private void enableNurseConsult() {
+		extensions.add(visitAction(Extensions.NURSE_CONSULT_NOTE_VISIT_ACTION,
```

Comment:
add to the list of "visit actions" a "nurse consult" visit action, which opens the form "nurseConsult.xml".  use the stethoscope icon; only display if the current location is tagged as a "Consult Note Location"; available to all users, no specific privileges required (we likely want to tweak this)